CMakeLists.txt

```diff
@@ -118,102 +130,36 @@ endif()
 string(REPLACE "include/" "${CMAKE_CURRENT_SOURCE_DIR}/include/" PYBIND11_HEADERS
                "${PYBIND11_HEADERS}")
 
-# Classic mode
-
-include("${CMAKE_CURRENT_LIST_DIR}/tools/pybind11Tools.cmake")
-
 # Cache variables so pybind11_add_module can be used in parent projects
 set(PYBIND11_INCLUDE_DIR
     "${CMAKE_CURRENT_LIST_DIR}/include"
     CACHE INTERNAL "")
-set(PYTHON_INCLUDE_DIRS
```

Comment:
For projects that currently depend on these being available as cached variables, will everything still continue to work?

Reply:
They still are, they are just moved to the pybind11Tools.cmake file, since they are Python discovery specific. This is also why we still default to the old discovery system, and the new one is opt-in - for the new system, these variables will not be available. (Plus it would be a CMake version-dependent switch, which is not a good idea usually).